06-Customizing AzTS for your org/Extending AzTS/ManagementGroupBasedComplianceInitiative.md

```diff
@@ -67,6 +67,34 @@ By default, this feature is not turned on. If you haven't enabled it in your AzT
 10. Click 'OK' and then click 'Save'.
 ![Resources](../../Images/06_MGCIEnableMGAndSave.png)
 
+-----
+
+- Using helper script
+  - Download the script from [here](../../TemplateFiles/DeploymentFiles.zip), skip this step if you have already downloaded the deployment package zip
+  > **Note:** Script can be downloaded by clicking Alt+Raw button.
+  - Open a PowerShell session.
+  - Navigate to the download location of the script in PowerShell session.
+    - cd "Script download location" 
```

Comment:
CD "<LocalExtractedFolderPath>\DeploymentFiles"
Refer this: https://github.com/azsk/AzTS-docs/tree/main/01-Setup%20and%20getting%20started#step-3-of-6-download-and-extract-deployment-package

Reply:
Done